### DISCUSSION TITLE
[Question] Where can I get the resources used to code this?

### DISCUSSION THREAD

**Discussion by Hexality** [berarma/oversteer#245] [Q&A]
Hello, I kinda wanted to ask if there's a wiki or something wiki references to everything used to develop this software. My plan would be to learn and try coding something similar for windows so I can revive my G27 without having to rely on the garbage official and deprecated Wingman Software by Logitech.

**Comment by berarma** [comment]
The code for this project depends completely on the drivers for Linux. To implement something like this you would need to know how to configure the Windows drivers for your Logitech wheel.

If you mean to replace the Windows drivers, [new-lg4ff](/berarma/new-lg4ff) contains more useful code and information.
